cypress/integration/console/devices/create-repository-registration.spec.js

```diff
@@ -154,49 +124,81 @@ describe.skip('End device repository create', () => {
+        cy.findByTestId('modal-window')
+          .should('be.visible')
+          .within(() => {
+            cy.findByText('Change input method', { selector: 'h1' }).should('be.visible')
+            cy.findByText(
+              'Are you sure you want to change the input method? Your current form progress will be lost.',
+            ).should('be.visible')
+            cy.findByRole('button', { name: /Change input method/ }).click()
+          })
+        cy.findByLabelText('Enter end device specifics manually').should('be.checked')
+      })
+
+      it('succeeds hiding provisioning information on JoinEUI reset', () => {
+        cy.findByLabelText('End device brand').selectOption('test-brand-otaa')
+        cy.findByLabelText('Model').selectOption('test-model2')
+
+        cy.findByLabelText('Frequency plan').selectOption('EU_863_870_TTN')
+        cy.findByLabelText('JoinEUI').type(generateHexValue(16))
+        cy.findByRole('button', { name: 'Confirm' }).click()
+        cy.findByLabelText('DevEUI').type(generateHexValue(16))
+        cy.findByLabelText('AppKey').type(generateHexValue(32))
+
+        cy.findByRole('button', { name: 'Reset' }).click()
+        cy.findByLabelText('DevEUI').should('not.exist')
+        cy.findByLabelText('AppKey').should('not.exist')
+      })
+
       it('succeeds registering device with single region, hardware and firmware versions', () => {
-        const devId = 'test-model2-device'
+        const devEui = generateHexValue(16)
 
         // End device selection.
-        cy.findByLabelText('Brand').selectOption('test-brand-otaa')
+        cy.findByLabelText('End device brand').selectOption('test-brand-otaa')
         cy.findByLabelText('Model').selectOption('test-model2')
 
         // End device registration.
         cy.findByLabelText('Frequency plan').selectOption('EU_863_870_TTN')
-        cy.findByLabelText('AppEUI').type(generateHexValue(16))
-        cy.findByLabelText('DevEUI').type(generateHexValue(16))
+        cy.findByLabelText('JoinEUI').type(generateHexValue(16))
+        cy.findByRole('button', { name: 'Confirm' }).click()
+        cy.findByLabelText('DevEUI').type(devEui)
         cy.findByLabelText('AppKey').type(generateHexValue(32))
-        cy.findByLabelText('End device ID').type(devId)
+        cy.findByLabelText('End device ID').should(
+          'have.value',
+          `eui-${devEui.toLocaleLowerCase()}`,
+        )
 
-        cy.findByRole('button', { name: 'Register end device' }).click()
+        cy.findByRole('button', { name: 'Add end device' }).click()
 
         cy.location('pathname').should(
           'eq',
-          `${Cypress.config('consoleRootPath')}/applications/${appId}/devices/${devId}`,
+          `${Cypress.config(
+            'consoleRootPath',
+          )}/applications/${appId}/devices/eui-${devEui.toLocaleLowerCase()}`,
         )
         cy.findByTestId('full-error-view').should('not.exist')
```

Comment:
You have to wait here until the page was actually loaded before checking whether an error is in the page.
```suggestion
        cy.findByRole('heading', { name: `eui-${devEui}` }).should('be.visible')
        cy.findByTestId('full-error-view').should('not.exist')
```

Please update other tests to do the same.